### DISCUSSION TITLE
Feature request: use encoding.TextMarshaler & encoding.TextUmarshaler

### DISCUSSION THREAD

**Discussion by muir** [99designs/gqlgen#1960] [Q&A]
### What happened?
I had to write UnmarshalGQL for something that already supported UnmarshalText.

### What did you expect?
gqlgen to make use of exiting implementations of `encoding.TextUmarshaler` and `encoding.TextMarshaler`.

### Minimal graphql.schema and models to reproduce
na

### versions
 - `gqlgen version v0.16.0`

**Comment by frederikhors** [comment]
Please add a github repo for reproduction.

**Reply by muir** [reply]
I don't have a public repo I can share for this.
